src/app/workspace/resource/representation/video/video.component.html

```diff
@@ -37,6 +54,25 @@
         </mat-toolbar-row>
 
         <mat-toolbar-row class="action">
+            <!-- vertical more button with menu to open and copy url -->
+            <button mat-icon-button [matMenuTriggerFor]="more">
+                <mat-icon>more_vert</mat-icon>
+            </button>
+            <mat-menu #more="matMenu">
+                <button mat-menu-item (click)="openVideoInNewTab(video['changingThisBreaksApplicationSecurity'])">
```

Comment:
I believe this will [break in production](https://stackoverflow.com/questions/56126968/angular-changingthisbreaksapplicationsecurity-gives-ts-error-property-chang).

You'll probably have to use the [bypass methods provided by Angular](https://angular.io/api/platform-browser/DomSanitizer).

Reply:
done in https://github.com/dasch-swiss/dsp-app/pull/798/commits/6ae945d5e1bcc630370e9f7135a731cb9d36645b